src/prpy/tsr/tsr.py

```diff
@@ -41,6 +48,8 @@ def __init__(self, T0_w=None, Tw_e=None, Bw=None,
             Tw_e = numpy.eye(4)
         if Bw is None:
             Bw = numpy.zeros((6, 2))
+        elif any((Bw[:, 1]-Bw[:, 0]) > 2*pi + EPSILON):
+            raise(ValueError('Bw range must be within 2*PI'))
```

Comment:
So, if a bound straddles pi, can't I just transform `T0_w` about that axis by pi and rotate the bounds by pi so that they straddle 0?